numba/tests/test_lists.py

```diff
@@ -692,8 +693,10 @@ def test_mul_error(self):
         pyfunc = list_mul
         cfunc = jit(nopython=True)(pyfunc)
         # Fail in malloc()
-        with self.assertRaises(MemoryError):
+        with self.assertRaises(MemoryError) as cm:
             cfunc(1, 2**58)
+        libc = ct.CDLL('libc.dylib')
+        libc.printf("###Please ignore above error message i.e. can't allocate region. It is in fact purpose of the this test to overflow memory###\n".encode("UTF-8"))
```

Comment:
These two lines need to only run on OSX (and won't work else where because there's no `libc.dylib` library present anywhere else). Once that is done CI should pass.

Reply:
Thanks @stuartarchibald 
I missed that. This fix is OSX specific. 
I am planning to use below code to identify OSX platform and run above two lines accordingly.
>>> import platform
>>> platform.system()
'Darwin'

Sorry i missed CI build. I didn't know about it. 
However i figured it out and initiate build on 
https://travis-ci.org/
Just wanted to check if thats the correct approach.